database.rb

```diff
@@ -0,0 +1,15 @@
+require 'json'
+
+module Database
+  def add_to_json(books)
+    File.write('./data/books.json', JSON.generate(books))
+  end
+
+  def save_people(people)
+    File.write('./data/people.json', JSON.generate(people))
+  end
+
+  def save_rentals(rentals)
+    File.write('./data/rentals.json', JSON.generate(rentals))
+  end
+end
```

Comment:
- [ ] while creating a rental I got the following error please make sure to fix it in order for your app will work properly 
![Screen Shot 2022-11-08 at 15 15 10](https://user-images.githubusercontent.com/61790587/200588116-91ddd915-cb35-4957-907a-ad009304285b.png)